doc/classes/ScriptEditor.xml

```diff
@@ -86,6 +86,7 @@
 			<argument index="1" name="base_path" type="String">
 			</argument>
 			<description>
+				Open the script create dialog. The script will extend base_name. The file extension can be omitted from base_path. It will be added based on the selected scripting language.
```

Comment:
```suggestion
				Opens the script create dialog. The script will extend [code]base_name[/code]. The file extension can be omitted from [code]base_path[/code]. It will be added based on the selected scripting language.
```

Reply:
Amended, thanks!

Reply:
What is the usual workflow after I make changes to an PR? Do I just "resolve" the conversation or do I also need to re-request a review (using github's button)?